controllers/C_Document.class.php

```diff
@@ -1341,5 +1352,24 @@ function _check_relocation($url, $new_pid = null, $new_name = null) {
 	return $url;
 }
 
+
+/**
+ * This function detects a MIME type for a file and check if it in the white list of the allowed mime types.
+ * @param string $file - file location.
+ * @param array $whiteList - array of mime types that allowed to upload.
+ */
+function isWhiteFile($file, $whiteList){
```

Comment:
Would there be a way to collect the whitelist within this function and migrate this function to the following script (which is always included in all scripts):
https://github.com/openemr/openemr/blob/master/library/sanitize.inc.php